tox.ini

```
@@ -25,6 +28,11 @@ commands=
     yamllint -s .
 
 [testenv:docs]
+description = Build docs
 deps = -r{toxinidir}/docs/requirements.txt
 commands =
     sphinx-build -E -W -d docs/build/doctrees -b html docs docs/build/html
```

Unrelated to this PR, I also recommend adding
```suggestion
    sphinx-build -E -n -W --keep-going -d docs/build/doctrees -b html docs docs/build/html
```